adal/src/main/java/com/microsoft/aad/adal/Logger.java

```diff
@@ -40,69 +38,125 @@
  */
 public class Logger {
     private static Logger sINSTANCE = new Logger();
-    static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
 
-    // Turn on the verbose level logging by default.
-    private LogLevel mLogLevel = LogLevel.Verbose;
     private ILogger mExternalLogger = null;
-    private static final String CUSTOM_LOG_ERROR = "Custom log failed to log message:%s";
-    private boolean mAndroidLogEnabled = false;
-    private String mCorrelationId = null;
 
-    // Disable to log PII by default.
-    private boolean mEnablePII = false;
+    private String mCorrelationId = null;
 
     /**
      * @return The single instance of {@link Logger}.
+     *
+     * @deprecated use {@link com.microsoft.identity.common.internal.logging.Logger#getInstance()} instead.
      */
+    @Deprecated
     public static Logger getInstance() {
         return sINSTANCE;
     }
 
     /**
      * Set the log level for diagnostic purpose. By default, the sdk enables the verbose level logging.
-     *
+     * @deprecated use {@link com.microsoft.identity.common.internal.logging.Logger#setLogLevel(com.microsoft.identity.common.internal.logging.Logger.LogLevel)} instead.
      * @param logLevel The {@link LogLevel} to be enabled for the diagnostic logging.
      */
+    @Deprecated
```

Comment:
same question here

Reply:
if you are expecting apps to call into common core, that is not an acceptable solution